spec/magma_file_attribute_spec.rb

```diff
@@ -0,0 +1,150 @@
+require_relative '../lib/magma'
+
+describe Magma::FileAttribute do
+  it 'stores a JSON database type' do
+    file_attribute = Magma.instance.magma_projects[:labors].models[:monster].attributes[:stats]
```

Comment:
You could replace this with

```ruby
file_attribute = Labors::Monster.attributes[:stats]
```

Not a huge difference but it saves a few keystrokes 😄